cms/static/sass/xmodule/_image-zoom-tool.scss

```diff
@@ -0,0 +1,64 @@
+// The LMS version is in:
+// /lms/static/sass/course/_image-zoom-tool.scss
+
+// edX image zoom tool
+.zooming-image-place {
```

Comment:
Each of these needs slightly different styles (i.e., the zoomed disc on the right versus left). I'm going to keep these here for now.